nodejs/aws-infra/examples/ec2/index.ts

```diff
@@ -0,0 +1,335 @@
+// Copyright 2016-2018, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as pulumi from "@pulumi/pulumi";
+
+import * as aws from "@pulumi/aws";
+import * as awsinfra from "@pulumi/aws-infra";
+
+import { Config, Output } from "@pulumi/pulumi";
+
+const network = awsinfra.Network.getDefault();
+const cluster = new awsinfra.x.Cluster("testing", { network });
```

Comment:
If I omit the network argument, does it use the default?

Reply:
I see later that, yes, it does.

Reply:
yup.  this was more to allow the test to serve more as an explicit example of how you might do things.